sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DeviceIdentity.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// Device identity.
+    /// </summary>
+    [CodeGenModel("Device")]
+    public partial class DeviceIdentity
```

Comment:
I'd rather go with IotDevice or HubDevice.

With DeviceIdentity, we'll end up with properties like DeviceIdentity.DeviceId, which is odd to me. Also, it is more than just identity.

Reply:
Hmm, interesting. I do like the sound of `IotDevice`.